alf/algorithms/mi_estimator.py

```diff
@@ -300,7 +300,7 @@ def calc_pmi(self, x, y, y_distribution=None):
         """Return estimated pointwise mutual information.
 
         The pointwise mutual information is defined as:
-            log P(x|y)/P(x) = log P(y|x)/P(y)
+            :math:`log P(x|y)/P(x) = log P(y|x)/P(y)`
```

Comment:
\log \frac{P(x|y)}{P(x)} = \log \frac{P(y|x)}{P(y)} as in latex is probably better